src/components/common/WalletIcon/index.tsx

```diff
@@ -49,14 +49,24 @@ const getWalletIcon = (provider: string) => {
   return WALLET_ICONS[label]
 }
 
-const WalletIcon = ({ provider, width = 30, height = 30 }: { provider: string; width?: number; height?: number }) => {
-  const icon = getWalletIcon(provider)
+const WalletIcon = ({
+  provider,
+  width = 30,
+  height = 30,
+  icon,
+}: {
+  provider: string
+  width?: number
+  height?: number
+  icon?: string
+}) => {
+  const svg = getWalletIcon(provider) || icon
```

Comment:
Maybe out of scope but I just tried inverting this and it works nicely (although there's a hydration issue).

Do you think it's still worth keeping `getWalletIcon`?

Reply:
Yeah, good point, we could kill it and not import all those icons if onboard handles that.

Reply:
Tested with different wallets, icons still work. ✅